tests/test_dtypes.py

```diff
@@ -218,6 +217,71 @@ def test_category_dtype_coerce():
     assert isinstance(validated_df, pd.DataFrame)
 
 
+def helper_type_validation(dataframe_type, schema_type, debugging=False):
+    """Helper function for using same or different dtypes for the dataframe and the schema_type"""
+    df = pd.DataFrame({"column1": [dataframe_type(1)]})
+    if debugging:
+        print(dataframe_type, df.column1)
+    schema = pa.DataFrameSchema({"column1": pa.Column(schema_type)})
+    if debugging:
+        print(schema)
+    schema(df)
+
+
+def test_numpy_type():
+    """Test various numpy dtypes"""
+    # Test correct conversions
+    valid_types = (
+        (np.complex, np.complex),  # Pandas converts complex numbers always to np.complex128
+        (np.complex, np.complex128),
+        (np.complex128, np.complex),
+        (np.complex64, np.complex128),  # Pandas converts complex numbers always to np.complex128
+        (np.complex128, np.complex128),
+        (np.float, np.float),
+        (np.float, np.float64),
+        (np.float16, np.float64),  # Pandas converts float numbers always to np.float64
+        (np.float32, np.float64),
+        (np.float64, np.float64),
+        (np.int, np.int),
+        (np.int, np.int64),
+        (np.int8, np.int64),   # Pandas converts int numbers always to np.int64
```

Comment:
is this supposed to be on line 246?

Reply:
I made multiple lines for a better overview.